my-app/src/dataItem.css

```diff
@@ -0,0 +1,104 @@
+.dataItemWrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  margin: auto;
+
+  border: solid 2px darkgray;
+  background-color: whitesmoke;
+  color:#313237;
+
+  font-family: 'Montserrat', sans-serif;
+
+  width: 280px;
+  height: 500px;
+
+  box-sizing: border-box;
```

Comment:
Move it to * selector and remove from all other cases.

Reply:
it is here only once